Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -441,7 +449,11 @@ def fromXML(self, name, attrs, content, ttFont):
 				if name != "pt":
 					continue  # ignore anything but "pt"
 				coordinates.append((safeEval(attrs["x"]), safeEval(attrs["y"])))
-				flags.append(not not safeEval(attrs["on"]))
+				flag = not not safeEval(attrs["on"])
+				if "overlap" in attrs:
+					f = safeEval(attrs["overlap"])
+					flag += f * flagReserved1
```

Comment:
better use "or" bitwise operator instead of "+" here: `flag |= f * flagReserved1`
let's rename `flagReserved1` to OVERLAP_SIMPLE